plinth/modules/security/__init__.py

```diff
@@ -42,6 +44,18 @@ def init():
     menu.add_urlname(title, 'glyphicon-lock', 'security:index')
 
 
+def setup(helper, old_version=None):
+    """Install the required packages"""
+    helper.install(managed_packages)
+    setup_fail2ban()
+
+
+def setup_fail2ban():
+    action_utils.service_unmask('fail2ban')
```

Comment:
This won't work directly from Plinth in normal usage case, because Plinth doesn't run as root user, and won't have privileges to unmask or enable a service.

Instead, we need to use actions.superuser_run (https://github.com/freedombox/Plinth/blob/master/plinth/actions.py#L112), and run the "service" action script (https://github.com/freedombox/Plinth/blob/master/actions/service) with appropriate parameters.